app/views/components/_nav_drawer.erb

```diff
@@ -0,0 +1,31 @@
+<drawer class="usa-width-one-fourth drawer" aria-hidden="true">
```

Comment:
I believe this file is unused (removed in https://github.com/18F/micropurchase/pull/583)